### PR TITLE
chore: remove Zend Diactoros

### PR DESCRIPTION
**Pull request by misaert**
This package is abandoned since `2019-12-31`.

### CODE REVIEW

**Comment by misaert** [comment]
Hi @bpolaszek,

Are you ok with this PR? [Another one](https://github.com/misaert/uri-factory/pull/1) is ready to update required PHP version to 8.0 and other packages after.

File: src/UriFactory.php [hunk 1]
```diff
@@ -58,7 +56,7 @@ public function createUri(string $uri = '', UriFactoryInterface $factory = null)
             }
         }
         throw new \RuntimeException(
-            "No adapter is installed. Please install guzzlehttp/psr7, zendframework/zend-diactoros or league/uri."
+            "No adapter is installed. Please install guzzlehttp/psr7 or league/uri."
```
Comment (bpolaszek) [inline src/UriFactory.php]: We added more adapters over time, let's advertise them here:

> No adapter is installed. Please install nyholm/psr7,  ringcentral/psr7, guzzlehttp/psr7 or league/uri.

**Review by bpolaszek — CHANGES_REQUESTED** [review]

**Comment by misaert** [comment]
Done :+1:

**Review by bpolaszek — APPROVED** [review]

**Comment by bpolaszek** [comment]
Thanks dude! 👍

**Pull request merged**